### ISSUE TITLE
Request (Size in MB and Keyboard Shortcuts)

### ISSUE THREAD

**Issue by blofstedt** [photopea/photopea#640]
Can you add an option to make keyboard shortcuts to specific functions? Also, could you please add the ability to see how many MB the current image you're working on is in the bottom of the screen somewhere? This would be useful for me as I make web adverts and try to keep the images under 1MB.

Thanks!

**Comment by photopea** [comment]
Hi. We only have fixed keyboard shortcuts right now, you can not change them. Would you like to add a new shortcut, or change any current one?

MB: What should these MB mean? Do you need to know the size of an exported PNG / JPG? Their size usually depends on the amount of compression, that you use during the export.

**Comment by blofstedt** [comment]
Hi! Please add a fixed shortcut for the trim function. 

Also, in terms of the MB, what I usually do in Photoshop is go to "image size" (I'm pretty sure that's what it's called, I'm not in front of a computer right now) and Photoshop gives you three options: the x and y axis dimensions in inches the and DPI. For web images, I usually set the inches to 4.5 by 6 or something along those lines and I can see the megabytes change in real time in that screen so I know I'm at 1mb or less. If I've changed the inches and the image is still over a mb, I may change the DPI to around 75. That way the image is still sharp for web use, but it is not larger than one megabyte.

Lastly, in Photoshop, you can see the size of an image in megabytes at the bottom of the screen as you're working on the image. I'd absolutely love it if you guys could add this feature as it frequently saves me time (if I see the image is under 1mb at the bottom of the screen, I just save the image and don't have to bother resizing).

Thanks!

**Comment by photopea** [comment]
Hi, I added the Trim function under the shortcut Ctrl + .  (Control and Dot).

I think you should try to work with pixels instead of inches. At the end, inches are converted to pixels by multiplying Inches and DPI ( 6 inches x  75 DPI = 450 pixels) - even when Photoshop still displays all dimensions in Inches.

The size in MB does not mean anything. It is just the image area times 3.  E.g. when your document is 400 x 600 pixels, it will show you 400x600x3 = 720 000 Bytes = 0.72 MB.

**Comment by blofstedt** [comment]
Thanks for adding the shortcut! But is it possible to add a way to see the megabytes? If you don't want to that's ok... I just find it useful, that's all.

**Comment by photopea** [comment]
I don't think it is useful. This size in MB does not have any meaning. It is not the size of PSD or the size of output graphics. Why exactly do you use 1 MB as a meaningful value?

**Comment by blofstedt** [comment]
I see what you mean. I've been wrong all this time! You're right, that megabyte figure is completely useless... 
I'm sorry if my questions are a bit silly, but is the JPG: B when I'm saving the image the number of bytes in the image?
